python/ray/dag/tests/experimental/test_accelerated_dag.py

```diff
@@ -267,6 +267,39 @@ def test_actor_method_bind_same_constant(ray_start_regular):
     compiled_dag.teardown()
 
 
+def test_actor_method_bind_same_arg(ray_start_regular):
```

Comment:
For tests of the new behavior, can you also check behavior when we pass different values to `dag.execute`? I.e. check that different executions are picking up the correct cached values.

Reply:
Updated the tests to run `execute()` multiple times with different value. Let me know if you suggested more than this.